scripts/BLmotors

```diff
@@ -0,0 +1,311 @@
+                exConfig = pm.diff_config(motors[motor])
+                if exConfig:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append(motorPtwo)
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("No")
+                else:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append(motorPtwo)
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("Yes")
+
+            except Exception:
+                infoperRecord.append(motors[motor])
+                infoperRecord.append(motorPtwo)
+                infoperRecord.append("None")
+                infoperRecord.append(" ")
+        except Exception:
+            try:
+                mconf = pm.get_config(motors[motor])
+                exConfig = pm.diff_config(motors[motor])
+                if exConfig:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append("No found!")
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("No")
+                else:
+                    infoperRecord.append(motors[motor])
+                    infoperRecord.append("No found!")
+                    infoperRecord.append(mconf)
+                    infoperRecord.append("Yes")
+            except Exception:
+                infoperRecord.append(motors[motor])
+                infoperRecord.append("No found!")
+                infoperRecord.append(" ")
+                infoperRecord.append(" ")
+        totalinfo.append(infoperRecord)
+    return totalinfo
+
+
+def displayfullInfo(totalinfo, argNoconf, argMatch,
+                    argFound, argsaliasConf, cfg):
+    """
+    This function displays in a table the information
+    of all the ims motors by selection of the user.
+    """
+    noConf = [info for info in totalinfo if info[0] == info[1]
+              and info[2] == "None" or info[0] != info[1]
+              and info[3] != "None" and info[1] != "No found!"]
+    noMatchConf = [info for info in totalinfo if info[3] == "No"]
+    noFound = [info for info in totalinfo if info[1] == "Not found!"]
+    aliasConf = [info for info in totalinfo if info[0] != info[1]
+                 and info[1] != "No found!" and info[2] != "None"]
+
+    if argNoconf:
+        if noConf:
+            print('Motors with no Configuration')
+            sortedListNoconf = sorted(noConf, key=lambda x: x[1])
+            tableNoconf = PrettyTable()
+            tableNoconf.field_names = ["Record Name", "Pv Name",
+                                       "Configuration",
+                                       "Configuration Matches"]
+            tableNoconf.add_rows(sortedListNoconf)
+            print(tableNoconf)
+        else:
+            print('The IMS motors have a configuration or/and',
+                  ' the IMS motors were not found!')
+
+    elif argMatch:
```

Comment:
The `elif` means that only one of the options can be used at a time, right? I suggest either changing these all to `if` so the user can see multiple reports at once or writing in the help text that only one can be used at a time.